repo/packages/K/kafka/6/config.json

```diff
@@ -0,0 +1,506 @@
+{
+  "type":"object",
+    "properties":{
+      "service":{
+        "type":"object",
+        "properties":{
+          "role":{
+            "default":"*",
+            "type":"string"
+          },
+          "principal":{
+            "default":"",
+            "type":"string"
+          },
+          "secret":{
+            "type":"string",
+            "description":"Mesos secret (password)",
+            "default":""
+          },
+          "user":{
+            "default":"root",
+            "type":"string"
+          },
+          "placement-strategy":{
+            "description":"Broker placement strategy. See documentation. [ANY, NODE]",
+            "type":"string",
+            "default":"NODE"
+          },
+          "plan-strategy":{
+            "description":"Broker placement strategy. See documentation. [INSTALL, STAGE]",
+            "type":"string",
+            "default":"INSTALL"
+          }
+        },
+        "required":[
+          "placement-strategy",
+          "plan-strategy",
+          "role",
+          "principal",
+          "user"
+        ]
+      },
+      "brokers":{
+        "type":"object",
+        "properties":{
+          "cpus":{
+            "description":"Broker cpu requirements",
+            "type":"number",
+            "default":1.0
+          },
+          "mem":{
+            "description":"Broker mem requirements",
+            "type":"integer",
+            "default":512
+          },
+          "disk":{
+            "description":"Broker disk requirements (only respected with persistent volumes)",
+            "type":"integer",
+            "default":1000
+          },
+          "count":{
+            "type":"number",
+            "default":1
+          }
```

Comment:
The above defaults have been changed:

- `mem` (from 2048 to 512)
- `disk` (from 5000 to 1000)
- `count` (from 3 to 1)